src/draco/tools/fuzz/build.sh

```diff
@@ -19,7 +19,7 @@
 cmake $SRC/draco
 # The draco_decoder and draco_encoder binaries don't build nicely with OSS-Fuzz
 # options, so just build the Draco shared libraries.
```

Comment:
Please drop this comment about draco_decoder and draco_encoder if this is working now.

Reply:
Actually I just tested this and it seems to fail for non-AFL builds still.
Since this is going to be a more involved fix, do you think you all can look at why the OSS-Fuzz build fails ([logs](https://oss-fuzz-build-logs.storage.googleapis.com/log-6f5f4441-af18-4dd2-877d-3d69c812964b.txt)).
Either way, I'll close this PR.